doc/ref/plotting_options/size_layout.ipynb

```diff
@@ -0,0 +1,366 @@
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "(\n",
+    "      df.hvplot.scatter(x='bill_length_mm', y='bill_depth_mm',\n",
+    "                        frame_width=200, title=\"Default aspect\")\n",
+    "    + df.hvplot.scatter(x='bill_length_mm', y='bill_depth_mm',\n",
+    "                        frame_width=200, aspect=0.6, title=\"3:5 ratio\")\n",
+    "    + df.hvplot.scatter(x='bill_length_mm', y='bill_depth_mm',\n",
+    "                        frame_width=200, aspect='square', title=\"Square plot (1:1 ratio)\")\n",
+    "    + df.hvplot.scatter(x='bill_length_mm', y='bill_depth_mm',\n",
+    "                        frame_width=200, aspect='equal', title=\"Re-computed frame_height\")\n",
+    ").cols(2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "24837128-7483-4a70-9fa0-011dc1d73606",
+   "metadata": {},
+   "source": [
+    ":::{seealso}\n",
+    "[`frame_width/frame_height`](option-frame_width__frame_height)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de030cee-46a2-4d8d-b12a-e685bd616e27",
+   "metadata": {},
+   "source": [
+    "(option-data_aspect)=\n",
+    "## `data_aspect`\n",
+    "\n",
+    "The `data_aspect` option sets the scaling between axis units, i.e., the ratio of the number of y-units per x-unit.\n",
+    "This is particularly useful for geographic or image data where you want equal distances on both axes (e.g., a degree of latitude and longitude should appear equal).\n",
+    "\n",
+    "Unlike [`aspect`](option-aspect) which controls **plot shape**, `data_aspect` affects the **visual scaling of the data itself**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eb20eb44-6216-4a94-94e7-1c50c2755eb0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.xarray  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "ds = hvsampledata.air_temperature(\"xarray\").sel(time=\"2014-02-25 12:00\")\n",
+    "ds.hvplot.image(data_aspect=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c99dceda-b76f-450a-90ac-06735a1b001c",
+   "metadata": {},
+   "source": [
+    "(option-frame_width__frame_height)=\n",
+    "## `frame_width / frame_height`\n",
+    "\n",
+    "The `frame_width` and `frame_height` options determine the width and height of the data area within the plot. They define the size of the plot’s core region (excluding axes, legends, and margins), allowing precise control over how the data is displayed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "de201f6b-0f7c-431a-8141-b86828068548",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "df.hvplot.scatter(\"bill_length_mm\", \"bill_depth_mm\", by=\"species\",\n",
```

Comment:
I'm not a big fan of this kind of formatting (I think I'm influenced by formatters like black and ruff)
<img width="618" alt="image" src="https://github.com/user-attachments/assets/972a2ced-f261-48cf-b131-d58eaf59a623" />

I much prefer this:
<img width="468" alt="image" src="https://github.com/user-attachments/assets/24172b0d-f1e9-48e5-813a-0643810ec59c" />